gallery/visualtests/nxCheckbox.js

```diff
@@ -20,64 +29,43 @@ describe('NxCheckbox', function() {
-      const blurSelector = `${selector} input`,
-          [targetElement, blurElement] = await Promise.all([browser.$(selector), browser.$(blurSelector)]);
+      const inputSelector = `${selector} input`;
+
+      const [targetElement, inputElement] = await waitAndGetElements(selector, inputSelector);
 
-      await targetElement.scrollIntoView({ block: 'center' });
       await targetElement.click();
-      await browser.execute(function(el) {
-        el.blur();
-      }, blurElement);
-
-      try {
-        await browser.eyesRegionSnapshot(null, Target.region(targetElement));
-      }
-      finally {
-        // click again to reset the state
-        await targetElement.click();
-      }
+      await blurElement(inputElement);
+
+      await checkScreenshot(targetElement);
     });
 
     it(`has a blue background, white checkmark, a light blue outer border,
       and glow when clicked and focused`, async function() {
       const focusSelector = `${selector} input`,
-          [targetElement, focusElement] = await Promise.all([browser.$(selector), browser.$(focusSelector)]);
+          [targetElement, focusElement] = await waitAndGetElements(selector, focusSelector);
 
-      await targetElement.scrollIntoView({ block: 'center' });
       await targetElement.click();
-      await targetElement.moveTo({ xOffset: -10, yOffset: -10 });
-      await browser.execute(function(el) {
-        el.focus();
-      }, focusElement);
-
-      try {
-        await browser.eyesRegionSnapshot(null, Target.region(targetElement));
-      }
-      finally {
-        // click again to reset the state
-        await targetElement.click();
-      }
+      await moveMouseAway();
+      await focusElement.focus();
+
+      await checkScreenshot(targetElement);
     });
 
     it(`has a blue background and white checkmark with a light blue outer border
       when clicked, focused, and hovered`, async function() {
       const focusSelector = `${selector} input`,
-          [targetElement, focusElement] = await Promise.all([browser.$(selector), browser.$(focusSelector)]);
+          [targetElement, focusElement] = await waitAndGetElements(selector, focusSelector);
+          //targetElementBox = targetElement.boundingBox(),
+          //targetElementCenter = {
+            //x: targetElementBox.x + (targetElementBox.width / 2),
+            //y: targetElementBox.y + (targetElementBox.height / 2)
+          //};
 
-      await targetElement.scrollIntoView({ block: 'center' });
       await targetElement.click();
-      await targetElement.moveTo();
-      await browser.execute(function(el) {
-        el.focus();
-      }, focusElement);
-
-      try {
-        await browser.eyesRegionSnapshot(null, Target.region(targetElement));
-      }
-      finally {
-        // click again to reset the state
-        await targetElement.click();
-      }
+      await focusElement.focus();
+      await targetElement.hover();
+
+      await checkScreenshot(targetElement);
```

Comment:
Do we not need to reset by clicking away?

Reply:
> Do we not need to reset by clicking away?

Observe you `setupBrowser` works.  Each test gets a fresh page via `beforeEach`